packages/web-components/fast-foundation/src/disclosure/disclosure.spec.md

```diff
@@ -0,0 +1,119 @@
+# Disclosure
+
+## Overview
+
+A disclosure component is the combination of a button and section of content, a button that toggles the visibility of the section. As defined by the W3C:
```

Comment:
Nit on this intro - with the updated spec to follow the disclosure pattern, does this still make sense?

Reply:
Indeed, it required some changes. Updated. :)